docs/slimbootloader.md

```diff
@@ -100,3 +100,7 @@ bzImage bzImage-5.4.52-linux-intel-acrn-sos EFI loader sbl_os
 
 * PREGENERATED_SIGNING_KEY_SLIMBOOT_KEY_SHA256  
   Set the path to the image signing key. Default is "${TOPDIR}/cert/TestSigningPrivateKey.pem"
+
+* BB_CURRENT_MC
+  Get the Current Configuration using this variable when multiple configuration have specified in conf/local.conf using BBMULTICONFIG variable.
```

Comment:
It should be like

* BB_CURRENT_MC  
  Set multiconfiguration dependencies i.e BB_CURRENT_MC  = "sos"
  mcdependency  = "mc:sos:uos:recipe_name:task_on_which_to_depend"

Ref: https://www.yoctoproject.org/docs/current/mega-manual/mega-manual.html#dev-enabling-multiple-configuration-build-dependencies

Reply:
we set the multiconfiguration as BBMULTICONFIG = "sos vm0 vm1" and get the current configuration using the BB_CURRENT_MC variable.   Not sure if its the right approach. However, can you provide the right wordings to maintain the consistency with other variables definition in the readme file.

Reply:
You are right. I overlooked BB_CURRENT_MC variable.